.github/workflows/README.md

```diff
@@ -0,0 +1,5 @@
+# Workflows
+
+Replacing [Travis CI](https://travis-ci.org/) as this project's primary CI/CD tool, [GitHub Actions](https://help.github.com/en/actions) uses workflows to perform operations when specific events within the repository occur. Workflows are specified in YAML and can utilize shared components from other GitHub Actions to form expansive  pipelines.
```

Comment:
Sorry for the nit, but as per https://sembr.org/ could you break separate sentences / clauses onto separate lines?